Intersect.Editor/Localization/Strings.cs

```diff
@@ -1783,7 +1783,7 @@ public struct EventCommandList
 
             public static LocalizedString waitforroute = @"Wait for Move Route Completion of {00}";
 
-            public static LocalizedString warp = @"Warp Player [Map: {00} X: {01} Y: {02} Dir: {03}]";
+            public static LocalizedString warp = @"Warp Player [Map: {00} X: {01} Y: {02} Dir: {03} Fade: {04}]";
```

Comment:
I could not get this to work - any thoughts would be appreciated. It just does not show the new addition at all.

Reply:
Delete your editor strings file and have it regenerate or edit the changes in manually. 

Unfortunately changed lines don't regenerate. Only new lines do.

Reply:
Ah, right - makes sense, thanks!

Reply:
(For what it's worth - does in fact work)